lib/redshift_serverless_namespace_stack.py

```diff
@@ -14,15 +16,17 @@ def __init__(self, scope: cdk.Construct, construct_id: str, **kwargs) -> None:
         redshift_full_command_access = iam.ManagedPolicy.from_aws_managed_policy_name(
             "AmazonRedshiftAllCommandsFullAccess")
 
-        role = iam.Role(self, "sls-test-role",
+        role = iam.Role(self, "LMD20RedshiftServerlessRole",
                         assumed_by=iam.ServicePrincipal("redshift.amazonaws.com"),
                         managed_policies=[redshift_full_command_access])
 
+        secret = secretsmanager.Secret(self, "LMD20RedshiftPassword")
+
         namespace_configuration = {
             "namespace_name": namespace_name,
-            "admin_username": "master",
-            "admin_user_password": "xuL9cMx09#iE",
-            "db_name": "liberia",
+            "admin_username": REDSHIFT_DEFAULT_USER,
+            "admin_user_password": secret.secret_value,
```

Comment:
@bwighane did you modify the secret value file? if so then you'll have to run the configure_account_secrets.py file so that the secret is updated within AWS.

Reply:
@BranfordTGbieor The only files changed are outlined here https://github.com/Last-Mile-Health/lmd-aws-cdk-pipelines-datalake-infrastructure/pull/12/files. The secrete value will be generated on creation of the stack and stored in secretes manager. I will include that section in the documentation.

Reply:
I understand but we already have provision for that within the code. The "configure_account_secrets.py" pushes secrets to AWS while the "configuration.py" manages the reference to such secrets. Anyways, no worries, I will make the updates.